### PR TITLE
Increase Raw rows list limit to 10,000

### PR DESCRIPTION
**Pull request by erlendvollset**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ Changes are grouped as follows
 
 ## Unreleased
 
+## [2.4.3] - 2020-09-18
+- Increase raw rows lis limit to 10,000
```
Comment (me-ydv-5) [inline CHANGELOG.md:20]: Why is this needed though?

```suggestion
- Increase raw rows list limit to 10,000
```

**Review by me-ydv-5 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NilsBarlaug-cognite) [inline CHANGELOG.md:20]:
```suggestion
- Increase raw rows list limit to 10,000
```

Comment (erlendvollset) [reply CHANGELOG.md]: The changelog entry or the change?

Comment (me-ydv-5) [reply CHANGELOG.md]: the change

**Review by me-ydv-5 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (erlendvollset) [reply CHANGELOG.md]: speeeeeed 💨 10x rows in each request means 10x less network jumps

Comment (me-ydv-5) [reply CHANGELOG.md]: Alright, but does raw allow 10k rows retrieval?
